report_sale_nrq/i18n/ja.po

```diff
@@ -93,5 +93,5 @@ msgstr "単価"
 #. module: report_sale_nrq
 #: model:ir.ui.view,arch_db:report_sale_nrq.report_salequotation_document
 msgid "quotation"
-msgstr "quotation"
+msgstr "見積書"
```

Comment:
@TimLai125 Please revert this as well.  We don't want to have translation for this string.  This string is only used in logic `t-if` conditions in report templates.